README.md

```diff
@@ -0,0 +1,125 @@
+# TAMS Master Project 2022/2023
+
+This repository contains the codebase for a [Marimba](https://en.wikipedia.org/wiki/Marimba) playing robot developed during the TAMS Master Project 2022/2023. 
+It is currently still "work in progress".
+
+## Setup 
+
+The robot uses Ubuntu 20.04 and ROS noetic. 
```

Comment:
why not a newer version of Ubuntu?

Reply:
https://github.com/UHHRobotics22-23/robot_project/pull/2#issuecomment-1358061642